pkgs/top-level/python-packages.nix

```diff
@@ -5097,6 +5097,26 @@ in {
     };
   };
 
+  keyrings = {
+    alt = buildPythonPackage rec {
```

Comment:
This doesn't look good. It's supposed to be `keyrings-alt`. Don't nest it just so you get a dot.